CMakeLists.txt

```diff
@@ -86,6 +86,7 @@ tvm_option(USE_TENSORRT_CODEGEN "Build with TensorRT Codegen support" OFF)
 tvm_option(USE_TENSORRT_RUNTIME "Build with TensorRT runtime" OFF)
 tvm_option(USE_RUST_EXT "Build with Rust based compiler extensions, STATIC, DYNAMIC, or OFF" OFF)
 tvm_option(USE_VITIS_AI "Build with VITIS-AI Codegen support" OFF)
+tvm_option(USE_MLAS "Build with MLAS library" ON)
```

Comment:
I think we should have this as default OFF for now (also in config.cmake)

Reply:
Agree, I have made it OFF by default.